packages/esm-patient-registration-app/src/patient-registration/input/basic-input/select/select-input.test.tsx

```diff
@@ -29,4 +29,14 @@ describe.skip('select input', () => {
 
     await waitFor(() => expect(input.value).toEqual(expected));
   });
+  it('optional get added if the input is not required', () => {
```

Comment:
```suggestion

  it('should show optional label if the input is not required', () => {
```

@ayush-AI test names should needs be descriptive.